src/app/(app)/missing-shoes/components/MissingShoes.tsx

```diff
@@ -9,50 +9,33 @@ import {
   TableHeader,
   TableRow,
 } from "@/components/ui/table";
-import { MissingShoeStatus } from "@/constant/constant";
+import { MissingShoeStatus, ShoesTyes } from "@/constant/constant";
 import {
   ColumnDef,
   flexRender,
   getCoreRowModel,
   useReactTable,
 } from "@tanstack/react-table";
 import { Plus as PlusIcon } from "lucide-react";
-
-const shoes = [
-  {
-    color: "Red",
-    status: MissingShoeStatus.Missing,
-    size: 42,
-    ownerName: "John Doe",
-    ownerPhoneNumber: "+1234567890",
-    ownerAddress: "123 Main St, Springfield, IL",
-    time: "2024-12-09",
-    type:"formal"
-  },
-  {
-    color: "Blue",
-    status: MissingShoeStatus.Found,
-    size: 38,
-    ownerName: "Alice Smith",
-    ownerPhoneNumber: "+0987654321",
-    ownerAddress: "456 Oak Rd, Rivertown, TX",
-    time: "2024-12-08",
-    type:"formal"
-  },
-];
+import { MissingShoesStatusBadge } from "@/common/StatusBadge/MissingShoesStatusBadge";
+import { MissingShoesActions } from "../add/components/MissingShoesActions";
 
 type MissingShoes = {
+  id: number;
   color: string;
-  status: string;
-  size: number;
+  status: MissingShoeStatus;
+  size: string;
```

Comment:
Let's get rid of that. Description should have these details.

Reply:
Okay Sir I have this